test/conformance/tests/test_sriov_operator.go

```diff
@@ -1722,6 +1722,72 @@ var _ = Describe("[sriov] operator", func() {
 			})
 		})
 
+		Context("Daemon reset with shutdown", Ordered, func() {
+			var testNode string
+			var policy *sriovv1.SriovNetworkNodePolicy
+			resourceName := "resetresource"
+
+			BeforeAll(func() {
+				isSingleNode, err := cluster.IsSingleNode(clients)
+				Expect(err).ToNot(HaveOccurred())
+				if isSingleNode {
+					Skip("test not supported for single node")
+				}
+
+				sriovInfos, err := cluster.DiscoverSriov(clients, operatorNamespace)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(len(sriovInfos.Nodes)).ToNot(BeZero())
+				testNode = sriovInfos.Nodes[0]
+				iface, err := sriovInfos.FindOneSriovDevice(testNode)
+				Expect(err).ToNot(HaveOccurred())
+
+				policy, err = network.CreateSriovPolicy(clients, "test-policy-", operatorNamespace, iface.Name, testNode, 5, resourceName, "netdevice")
+				Expect(err).ToNot(HaveOccurred())
+				WaitForSRIOVStable()
+			})
+
+			It("should remove interface configuration from host even after reboot", func() {
+				By("force rebooting the node")
+				_, errOutput, err := runCommandOnConfigDaemon(testNode, "chroot", "/host", "reboot")
+				Expect(err).ToNot(HaveOccurred(), errOutput)
+				By("removing the policy")
+				err = clients.Delete(context.Background(), policy)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("waiting for node to be not ready")
+				Eventually(func(g Gomega) bool {
+					node, err := clients.CoreV1Interface.Nodes().Get(context.Background(), testNode, metav1.GetOptions{})
+					g.Expect(err).ToNot(HaveOccurred())
+					for _, con := range node.Status.Conditions {
+						if con.Type == corev1.NodeReady && con.Status != corev1.ConditionTrue {
+							return true
+						}
+					}
+					return false
+				}, waitingTime, 1*time.Second).Should(BeTrue())
```

Comment:
Can you refactor this to something like:
```
waitForNodeToBeReady()
waitForNodeToBeNotReady()
```
or
```
waitForNode(corev1.NodeReady, corev1.ConditionTrue)
waitForNode(corev1.NodeReady, corev1.ConditionFalse)
```

It took me a while to spot the differences between the calls